api/repositories/service_binding_repository_test.go

```diff
@@ -656,10 +717,10 @@ var _ = Describe("ServiceBindingRepo", func() {
 
 	Describe("ListServiceBindings", func() {
```

Comment:
The PR introduces the capability of filtering bindings by type. However, there is no test to verify that. Could you add one?